FabricLogging.py

```diff
@@ -50,10 +52,12 @@ def getLevelName(level):
         return result
     return f"Level {level}"
 
+
 class Filter():
     def __init__(self, name):
         self.name = name

```

Comment:
This class has just been left in as a mistake but is intended for future implementation.

The idea is that because we might define handlers over time i want to keep filters as a central class so that no matter what handler you are using you can use the same syntax for your filters.